pilot/pkg/model/context.go

```diff
@@ -218,6 +220,15 @@ func ApplyMeshConfigDefaults(yaml string) (*meshconfig.MeshConfig, error) {
 			return nil, multierror.Prefix(err, "failed to convert to proto.")
 		}
 	}
+
+	// Backward compat option: if mixer address is set but
+	// mixer_check_server and mixer_report_server are unset, copy the value
+	// into these two config vars.
+	if out.MixerAddress != "" && out.MixerCheckServer == "" && out.MixerReportServer == "" {
+		out.MixerCheckServer = out.MixerAddress
+		out.MixerReportServer = out.MixerAddress
+	}
+
```

Comment:
@costinm the code above should provide the requisite backward compatible behavior.

Reply:
Thanks, that's all we need.